lerna.json

```diff
@@ -1,7 +1,6 @@
 {
   "lerna": "2.0.0-rc.5",
-  "packages": [
-    "packages/*"
```

Comment:
Hey @matuzalemsteles 

Will removing this property affect the lerna publish task at all?

Reply:
Hey @Robert-Frampton,

When we enable yarn workspace in `lerna.json`:
```
{
  "npmClient": "yarn",
  "useWorkspaces": true
}
```

It will use the wire to initialize the project and will also use the `package.json/workspaces` field to find the packages instead of `lerna.json/packages`. I do not think this would have affected the publication.